web/app/js/components/Navigation.jsx

```diff
@@ -401,14 +413,24 @@ class NavigationBase extends React.Component {
             open={this.state.namespaceMenuOpen}
             keepMounted
             onClose={this.handleNamespaceMenuClick}>
+
+            <MenuItem>
+              <InputBase
+                id="namespace-filter-textfield"
+                value={namespaceFilter}
```

Comment:
What do you think about highlighting the selected namespace in the menu? Perhaps, from the user experience point of view, is useful.

```
<MenuItem
    selected={formattedNamespaceName === "All Namespaces"}
    value="all"
    onClick={e => this.handleNamespaceChange(e, "_all")}>
    All Namespaces
</MenuItem>

{namespaces.map(ns => (
    <MenuItem
    selected={ns.name === formattedNamespaceName}
    onClick={e => this.handleNamespaceChange(e, ns.name)}
    key={ns.name}>
        {ns.name}
    </MenuItem>
))}
```

<img width="275" alt="menu" src="https://user-images.githubusercontent.com/1440981/68301209-c13caa80-009f-11ea-9eea-8fa5bfaf81f2.png">

Reply:
Interesting idea -- I'm going to leave as is for now, but if the button is redesigned in the future this may be good to suggest.